packages/scripts/config/webpack.config.js

```diff
@@ -89,20 +83,38 @@ const config = {
 	},
 	module: {
 		rules: [
-			{
+			// Sourcemap Loader is a complement to the `devtool` configuration,
+			// and thus is only necessary for development environments.
+			! isProduction && {
 				test: /\.js$/,
 				use: require.resolve( 'source-map-loader' ),
 				enforce: 'pre',
 			},
 			{
 				test: /\.js$/,
 				exclude: /node_modules/,
-				use: {
-					loader: require.resolve( 'babel-loader' ),
-					options: getBabelLoaderOptions(),
-				},
+				use: [
+					'thread-loader',
```

Comment:
over on Calypso, we found that the optimal number wasn't CPU-1, but more like CPU / 2. MIght be worth experimenting with a bit.

Reply:
> over on Calypso, we found that the optimal number wasn't CPU-1, but more like CPU / 2. MIght be worth experimenting with a bit.

Hey Ben, thanks for checking in. I didn't note it in the original comments, but I did tinker with this value slightly, though in my case I tried _increasing_ it to the exact number of CPUs, and had observed a degraded benefit. I hadn't considered lowering it, but will give it a try.

Is there a pull request or issue which I could reference for that and/or other settings you'd explored?

Reply:
@aduth https://github.com/Automattic/wp-calypso/pull/25456 and https://github.com/Automattic/wp-calypso/pull/26059